src/test/resources/features/ifs/Supression.feature

```diff
@@ -200,4 +233,44 @@ Feature: Suppression
       | 2021-03-05 | 2021-04-20 | 47           | 2.6          | 10                      | 150502             |             |      |             |
       | 2021-02-01 | 2021-02-03 | 2            | 2.6          | 24                      | 350049             |             |      |             |
       | 2021-02-04 | 2021-03-04 | 29           | 0.0          | 0                       | 350000             | LEGISLATIVE | 1    | COVID       |
-      | 2021-03-05 | 2021-07-06 | 124          | 2.6          | 24                      | 353091             |             |      |             |
\ No newline at end of file
+      | 2021-03-05 | 2021-07-06 | 124          | 2.6          | 24                      | 353091             |             |      |             |
+
+  Scenario: Suppression applied when multiple applying suppressions with multiple rule types exist.
+    Given suppression data has been created
+      | suppressionId | reason    | description | enabled | fromDate   | toDate     |
+      | 1             | MAINTRANS | desc-1      | true    | 2021-04-04 | 2021-05-04 |
+    And suppression rules have been created
+      | ruleId | mainTrans | suppressionIds |
+      | 1      | 1546      | 1              |
+      | 1      | 1535      | 1              |
+       Given suppression data has been created
+      | suppressionId | reason     | description | enabled | fromDate   | toDate     |
+      | 3             | PERIOD-END | desc-3      | true    | 2021-06-01 | 2021-06-20 |
+    And suppression rules have been created
+      | ruleId | periodEnd  | suppressionIds |
+      | 1      | 2019-05-20 | 3              |
+    Given suppression data has been created
```

Comment:
Should this not be 'And', as best practice says you should not be ideally using 'Given' twice in the same scenario.

Reply:
good point. thanks